styles.css

```diff
@@ -229,8 +257,12 @@ h4 {
   overflow: hidden;
 }
 
-.content-section h2 {
-  color: #6e979e;
+.content-section li+li{
```

Comment:
Any `li` following another `li` will have a bit more spacing than default for better legibility